substrate/client/statement-store/src/lib.rs

```diff
@@ -72,13 +73,14 @@ use std::{
 };
 
 const KEY_VERSION: &[u8] = b"version".as_slice();
-const CURRENT_VERSION: u32 = 1;
+const CURRENT_VERSION: u32 = 2;
```

Comment:
The version should not be changed if you require to purge the store any way.

Reply:
what if the node downgrades?